docs/sources/whatsnew/whats-new-in-v9-1.md

```diff
@@ -0,0 +1,188 @@
+---
+_build:
+  list: false
+aliases:
+  - /docs/grafana/latest/guides/whats-new-in-v9-1/
+description: Feature and improvement highlights for Grafana v9.1
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '9.1'
+  - release notes
+title: What's new in Grafana v9.1
+weight: -33
+---
+
+# What's new in Grafana v9.1
+
+We're excited to announce Grafana v9.1, with a variety of improvements that focus on Grafana's usability, performance, and security. Read on to learn about new options to share and embed dashboards, search and navigation enhancements, new panel options and additional authentication features. You can also find out more about new Single Sign-On and role-based access control options in Grafana Enterprise, and more. For details, see the complete [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+## OSS
+
+### Auth and security
+
+#### Grafana Service Accounts are generally available
+
+Service accounts are an evolution in machine access within Grafana. You can create multiple API tokens per service account with independent expiration dates, and temporarily disable a service account without deleting it. These benefits make Service Accounts a more flexible way for Terraform and other apps to authenticate with Grafana.
+
+Service accounts also work with [role-based access control]({{< relref "../enterprise/access-control/" >}}) in [Grafana Enterprise]({{< relref "../enterprise/" >}}): you can improve security by granting service accounts specific roles to limit the functions they can perform. Service accounts have been in beta since Grafana v8.5, and during that time we've improved the UI and migration path from API keys, and made it possible to add service accounts to teams and inherit team permissions. To learn more about Service Accounts, see the [documentation]({{< relref "../administration/service-accounts" >}}).
```

Comment:
Fixing a run-on sentence. 
```suggestion
Service accounts also work with [role-based access control]({{< relref "../enterprise/access-control/" >}}) in [Grafana Enterprise]({{< relref "../enterprise/" >}}). 
You can improve security by granting service accounts specific roles to limit the functions they can perform. 
Service accounts have been in beta since Grafana v8.5. 
During that time, we've improved the UI and migration path from API keys, made it possible to add service accounts to teams, and inherit team permissions. 
To learn more about Service Accounts, see the [documentation]({{< relref "../administration/service-accounts" >}}).
```